iocell.8.txt

```diff
@@ -1196,6 +1196,17 @@ PROPERTIES
     Default: off
     Source: cpuset(1)
 
+  volumes=/host/dir[:/jail/dir[:mount_options]]
+
+  Mounts and unmounts directories from the host into the jail root using
+  nullfs(5).
+  'mount_options' are passed direcly to the mount command and defaults to "rw"
+  if ommitted.
+  If '/jail/dir' is also ommitted then the same value as '/host/dir' will be
```

Comment:
you like python, others like perl. again, i do not agree and won't change this extremely convenient use of volumes because of your religion.